clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -202,7 +203,7 @@ public boolean matches(AbstractRequest body) {
 
         sender.run(time.milliseconds());  // Send AddOffsetsRequest
         assertTrue(transactionManager.hasPendingOffsetCommits());  // We should now have created and queued the offset commit request.
-        assertFalse(addOffsetsResult.isDone());
+        assertTrue(addOffsetsResult.isCompleted());
```

Comment:
In this case (and only this case), the assertion was also changed from false to true. That's what I was asking about